content/efcore/getting-started/samples/generating-databases-and-data-models/generating-from-code-example/README.md

```diff
@@ -0,0 +1,9 @@
+# Primary Key
+
+This code example shows what the database schema would be for the model included in the example.
```

Comment:
This code example demonstrates how to generate a database from a data model and print its resulting schema through the `REPLService`.